vector/src/main/res/xml/vector_settings_general.xml

```diff
@@ -21,11 +22,18 @@
             android:summary="@string/change_password_summary"
             android:title="@string/settings_password" />
 
+        <im.vector.app.core.preference.VectorSwitchPreference
+            android:key="TCHAP_SETTINGS_HIDE_FROM_USERS_DIRECTORY_PREFERENCE_KEY"
+            android:summary="@string/tchap_settings_hide_from_users_directory_summary"
+            android:title="@string/tchap_settings_hide_from_users_directory_title" />
+
         <im.vector.app.core.preference.VectorPreference
+            android:enabled="false"
```

Comment:
Do not disable this field to display it correctly. Currently it is displayed in light grey instead of in primary color